components/financial_assistance/app/domain/financial_assistance/operations/applications/pvc/create_pvc_request.rb

```diff
@@ -56,16 +56,76 @@ def create_non_esi_evidences(application)
             end
```

Comment:
create_success_history is basically creating the record history unlike the other call, where we are creating the history when the evidence was created

Here we are creating the history _after_ the request was successfully published because the record history was stating "PVC - Renewal verifications submitted"

Let me know if this makes sense

Reply:
yes , but this step should be done right before validation.
- Which means that we started the PVC process 
**Failed case:** 
- failed due to some reason and record history
**Success case:**
- we will not record anything here , and the state will be in pending until a response is received.